docs/review_process.md

```diff
@@ -0,0 +1,144 @@
+which might be inaccessible (GitHub API, docker hub, etc.) and may result in intermittent failures.
+So, what to do once `Unexpected error` was encountered? You may consider re-running the build by closing your pull request, waiting 15 seconds, and then re-opening it again.
+
+Sometimes it's necessary to restart the build several times.
+If an `Unexpected error` persists, tag [@jgsogo](https://github.com/jgsogo) and [@danimtb](https://github.com/danimtb) asking for the help with CI.
+Alternatively, just [open a new issue](https://github.com/conan-io/conan-center-index/issues/new/choose).
+
+## Avoiding conflicts
+
+It's recommended to rebase your changes on top of the master branch to avoid conflicts.
+Right now, neither GitHub itself nor conan-center-bot notify about merge conflicts, so it's the contributor's responsibility to periodically check for the conflicts.
+Obviously, PRs that have merge conflicts are never merged, and all the conflicts have to be resolved first.
+
+## Draft
+
+Draft pull requests are also never merged. The same applies for PRs with the `WIP` keyword (stands for `Work in Progress`) in the title - GitHub considers them drafts as well.
+Once you're done with your changes, remember to convert from "Draft" to "Normal" pull request and remove WIP keyword from the title.
+
+## Getting your pull request reviewed
+
+Each PR must be reviewed by several reviewers before it will be merged. It cannot be just reviews from random people, we have two categories of reviewers:
+
+### Official reviewers
+
+The list includes only official Conan developers:
+
+- [@memsharded](https://github.com/memsharded)
+- [@lasote](https://github.com/lasote)
+- [@danimtb](https://github.com/danimtb)
+- [@jgsogo](https://github.com/jgsogo)
+- [@czoido](https://github.com/czoido)
+- [@solvingj](https://github.com/solvingj)
+- [@sse4](https://github.com/sse4)
+- [@uilianries](https://github.com/uilianries)
+
+### Community reviewers
+
+The list includes conan-center-index contributors who are very active and proven to be trusted - they frequently submitted pull requests and provide their own useful reviews:
+
+- [@madebr](https://github.com/madebr)
+- [@SpaceIm](https://github.com/SpaceIm)
+- [@ericLemanissier](https://github.com/ericLemanissier)
+- [@prince-chrismc](https://github.com/prince-chrismc)
+- [@Croydon](https://github.com/Croydon)
+- [@intelligide](https://github.com/intelligide)
+- [@theirix](https://github.com/theirix)
+- [@gocarlos](https://github.com/gocarlos)
+
+The list is not constant and will change periodically based on contribution.
+That also means you can be included in this list as well - submit PRs and provide reviews, and in time you may be added as a trusted contributor.
+
+### Rule of 3 reviews
+
+At least 3 reviews are required for an approval, and at least one of them has to be from the official reviewers.
+So, it might be 1 official + 2 community, or 3 official, but it couldn't be just 3 community reviews.
```

Comment:
```suggestion
So, it might be 1 official + 2 community, or 3 official, but it couldn't be just 3 community reviews.
Approvals are only counted if they are associated with the latest commit in the PR, while "Change requested" ones (from the Conan team) will persist even if there are new commits. Don't hesitate to dismiss old reviews if the issues have already been addressed.
```